### PR TITLE
Use OS default config dirs + minor fixes

### PR DESCRIPTION
**Pull request by spiarh**
* **Please check if the PR fulfills these requirements**
- [ ] The commit message follows the conventional commits guidelines
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

* **What kind of change does this PR introduce?** (Bug fix, feature, docs update, ...)

* **What is the current behavior?** (You can also link to an open issue here)

* **What is the new behavior (if this is a feature change)?**

* **Does this PR introduce a breaking change?** (What changes might users need to make in their application due to this PR?)

* **Other information**:

### CODE REVIEW

**Review by prit342 — APPROVED** [review]

File: internal/cli/config.go [hunk 1]
```diff
@@ -10,45 +11,55 @@ import (
...
 
-			if dev {
-				conf.Cli.Dev = true
-				conf.Container.Image = "controlplane/simulator:dev"
+	configCmd.Run = func(_ *cobra.Command, _ []string) {
+		if printDir {
+			dir, err := config.SimulatorDir()
+			cobra.CheckErr(err)
+			//nolint: forbidigo
+			fmt.Println(dir)
```
Comment (ricfeatherstone) [inline internal/cli/config.go:32]: Should this be slog?

File: internal/logging/factory.go [hunk 2]
```diff
@@ -8,7 +8,7 @@ import (
 func Configure() {
 	handlerOptions := &slog.HandlerOptions{
 		AddSource: true,
-		Level:     slog.LevelError,
+		Level:     slog.LevelInfo,
```
Comment (ricfeatherstone) [inline internal/logging/factory.go:11]: I think this should default to Error but with an env var to change the level
The CLI prints the output from the tools (packer, terraform, and ansible), and this is of interest to the user. Additional logging of what the CLI is doing is not of interest on the happy path

**Review by ricfeatherstone — COMMENTED** [review]
We need to discuss markdown auto formatting

```
- [//]: # (TODO document commands)
+ [//]: # "TODO document commands"

- |----------|-----------------------------------------------------------------------------------------------------------------|
+ | --------- | --------------------------------------------------------------------------------------------------------------- 
```

**Review by ricfeatherstone — APPROVED** [review]
LGTM

**Pull request merged**